jerry-core/config.h

```diff
@@ -115,6 +115,10 @@
 # define JERRY_BUILTIN_TYPEDARRAY JERRY_ESNEXT
 #endif /* !defined (JERRY_BUILTIN_TYPEDARRAY) */
 
+#ifndef JERRY_BUILTIN_SHAREDARRAYBUFFER
+# define JERRY_BUILTIN_SHAREDARRAYBUFFER JERRY_ESNEXT
+#endif /* !defined (JERRY_BUILTIN_SHAREDARRAYBUFFER) */
+
```

Comment:
JERRY_BUILTIN_SHAREDARRAYBUFFER should depend on JERRY_BUILTIN_TYPEDARRAY, it does not work otherwsie. We should have a check for this in this file.

Reply:
This is not resolved yet. See the end fo the file for such cross-dependency checks.